app/instrumentation-tests/src/org/commcare/androidTests/TilesTest.kt

```diff
@@ -0,0 +1,183 @@
+                .perform(click())
+        caseDetailsExpanded.areDisplayed()
+        Espresso.pressBack()
+        caseDetails.areDisplayed()
+        withText("Secret").doesNotExist()
+
+        // Confirm on pressing back we're in EntitySelectActivity
+        Espresso.pressBack()
+        val application = InstrumentationRegistry.getInstrumentation().targetContext.applicationContext
+                as CommCareInstrumentationTestApplication
+        var activity = application.currentActivity
+        InstrumentationUtility.assert(activity is EntitySelectActivity, "Current Activity is ${activity.localClassName}")
+
+        onView(withText("Sally Ride"))
+                .perform(click())
+        caseDetails.areDisplayed()
+        withId(R.id.com_tile_holder_btn_open).isDisplayed()
+
+        // Go to form screen
+        onView(withText("Placeholder"))
+                .perform(click())
+
+        withText("Inside").isDisplayed()
+        caseDetails.areDisplayed()
+
+        // Expand tile
+        onView(withId(R.id.com_tile_holder_btn_open))
+                .perform(click())
+        caseDetailsExpanded.areDisplayed()
+
+        //Close expanded list
+        onView(withId(R.id.com_tile_holder_btn_open))
+                .perform(click())
+        caseDetails.areDisplayed()
+        withText("Secret").doesNotExist()
+    }
+
+    @Test
+    fun testPersistentTile_withDetail() {
+        InstrumentationUtility.openModule(1)
+
+        // Clicking this opens all the case details including the expanded ones.
+        onView(withText("Sally Ride"))
+                .perform(click())
+        caseDetailsExpanded.areDisplayed()
+
+        // And we don't see the expand button
+        withId(R.id.com_tile_holder_btn_open).isNotDisplayed()
+
+        onView(withText("Continue"))
+                .perform(click())
+
+        caseDetails.areDisplayed()
+        withText("Placeholder").isDisplayed()
+        withId(R.id.com_tile_holder_btn_open).isNotDisplayed()
+
+        // Open form
+        onView(withText("Placeholder"))
+                .perform(click())
+
+        // We can still see the case details.
+        caseDetails.areDisplayed()
+        withId(R.id.com_tile_holder_btn_open).isNotDisplayed()
+    }
+
+    @Test
+    fun testPersistentTile_noDetail_noInline() {
+        InstrumentationUtility.openModule(2)
+
+        onView(withText("Sally Ride"))
+                .perform(click())
+        caseDetails.areDisplayed()
+        withText("Placeholder").isDisplayed()
+        withText("Secret").doesNotExist()
+        withId(R.id.com_tile_holder_btn_open).isNotDisplayed()
+
+        // go to form and you can still see the case details.
+        onView(withText("Placeholder"))
+                .perform(click())
+
+        caseDetails.areDisplayed()
+        withText("Secret").doesNotExist()
+        withId(R.id.com_tile_holder_btn_open).isNotDisplayed()
+    }
+
+    @Test
+    fun testBreadcrumb() {
```

Comment:
[This](https://github.com/dimagi/commcare-ui-tests/blob/master/features/tiles.feature#L184) and [that](https://github.com/dimagi/commcare-ui-tests/blob/master/features/tiles.feature#L194) are not present